README.md

```diff
@@ -29,7 +29,7 @@ Listening for messages coming from a kafka broker.
 ```json
 {
   "consumer" : {
-    "zookeeper.host" : "<zookeeperHost>",
+    "zookeeper.connect" : "<zookeeperConnectionString>",
     "port": "<zookeeperPort>",
```

Comment:
no, Should I send you another one or wait till next one (publish to multiple topics)?